e2e/pageobjects/ide/Ide.ts

```diff
@@ -229,6 +231,38 @@ export class Ide {
         await this.driverHelper.waitVisibility(selectedRightToolbarButtonLocator, timeout);
     }
 
+    async getApplicationUrl(notificationText: string) {
```

Comment:
On my opinion for avoiding possible confusion better rename to 
"getApplicationUrlFromNotification"